app/controllers/concerns/idv_step_concern.rb

```diff
@@ -27,6 +27,10 @@ def confirm_step_allowed
     redirect_to_fail_path if step_attempts_exceeded?
   end
 
+  def step_available
+    redirect_to account_url unless FeatureManagement.reveal_usps_flow?
+  end
```

Comment:
What do you think about restricting access at the route level instead, similarly to how we turned off LOA3? https://github.com/18F/identity-idp/blob/master/config/routes.rb#L115